src/vs/workbench/contrib/chat/browser/attachments/promptInstructions/promptInstructionsCollectionWidget.ts

```diff
@@ -66,6 +67,13 @@ export class PromptAttachmentsCollectionWidget extends Disposable {
 		return this.children.length === 0;
 	}
 
+	/**
+	 * Check if any of the attachments is a prompt file.
+	 */
+	public get hasPromptFile(): boolean {
+		return this.references.some((uri) => getPromptFileType(uri) === 'prompt');
```

Comment:
We allow to attach only the `instruction` files now, so this will always be false?